docs/windows.md

```diff
@@ -46,3 +46,18 @@ the explorer window by hitting `<cmd>+R` and type in:
 - `explorer %LOCALAPPDATA%\Programs\Ollama` contains the binaries (The installer adds this to your user PATH)
 - `explorer %HOMEPATH%\.ollama` contains models and configuration
 - `explorer %TEMP%` contains temporary executable files in one or more `ollama*` directories
+
+
+## Multi-user Advanced Scenarios
+
+The `OllamaSetup.exe` installer has been designed for individual use on your
+Windows PC.  It installs in your account without requiring Administrator rights.
+We update Ollama regularly to support the latest models, and this installer will
+help you keep up to date.
+
+If you have a multi-user system, or are looking for more control over where
```

Comment:
Maybe:

If you want to run Ollama as a system service because you have a multi-user system, or are looking for more control over where Ollama is installed, we provide a separate `ollama-windows-amd64.zip` file. This file contains the Ollama CLI and GPU library dependencies for Nvidia and AMD. It does not include the tray application and will not notify you when there is a new version of Ollama.